bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx109-en-features.html

```diff
@@ -0,0 +1,118 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-split') }}
+  {{ css_bundle('firefox_whatsnew') }}
+  {{ css_bundle('firefox_whatsnew_109_en_features') }}
+{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-whatsnew109-features' %}
+{% set _utm_campaign = 'whatsnew109' %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+  <h1 class="mzp-c-section-heading">Firefox helps you stay private and productive</h1>
+
+  {% call split(
```

Comment:
**Nit (blocking):** The split components are going off-screen in smaller screens

<img width="463" alt="Screen Shot 2023-01-12 at 5 32 19 PM" src="https://user-images.githubusercontent.com/42309026/212195500-05a0958f-a2c6-4046-ba66-71ed85a1f132.png">

Reply:
Also a bug in Protocol. Filed https://github.com/mozilla/protocol/issues/843.

I don't know if it needs to block this PR since this particular page gets pretty much zero mobile traffic but I'll see if it's an easy fix.